myapp/user.js

```diff
@@ -0,0 +1,35 @@
+  var obj = JSON.parse(usersJSON);
+  var l = obj.length;
+  var users = [];
+
+function User(data){
+  this.userName = data.userName;
+  this.email = data.email;
+  this.password = data.password;
+  this.logged=false;
```

Comment:
space after/before "=". Also boolean values are better named in the form "isSomething" or "hasSomething", eg isLoggedIn etc